libs/@local/hash-graph-sdk/python/graph_sdk/filter/__init__.py

```diff
@@ -0,0 +1,20 @@
+"""Ergonomic and type-safe filter paths.
+
+To start using this module, choose where you need to "start",
+depending on the query this will be either: `DataTypePath`,
+`PropertyTypePath`, `EntityTypePath`, or `EntityPath`.
```

Comment:
```suggestion
depending on the query this will be either: `DataTypeQueryPath`,
`PropertyTypeQueryPath`, `EntityTypeQueryPath`, or `EntityQueryPath`.
```